bitnami/deepspeed/templates/client/hostfile-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.client.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-hosts" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/part-of: deepspeed
+    app.kubernetes.io/component: client
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" (dict "value" .Values.commonAnnotations "context" $) | nindent 4 }}
+  {{- end }}
+data:
+  hostfile: |
+    {{- if .Values.overrideHostFile }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.config.overrideHostFile "context" $) | nindent 4 }}
+    {{- else }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.config.defaultHostFile "context" $) | nindent 4 }}
+    {{- end }}
```

Comment:
Not sure why and it isn't a big problem, when template is rendered the `hostfile` starts with a blank line

Reply:
You right, after installation CM shows the right content, helm template shows the blank line at the beginning. We can close this conversation, it shouldn't be a problem.